examples/secure-multi-party-computation/Duet/1-DS-1-DO/POC-MPCTensor-Duet-Party1-DS.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>MAYBE: The first link is not pointing to the correct address.</p><p>Also, could you remove the "ipynb" extension from the second one.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5020/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done. Thanks!</p>